app/commands/authenticate_user.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Authenticate a user with a jwt if they exist
+class AuthenticateUser
+  prepend SimpleCommand
+
+  def initialize(email, password)
+    @email = email
+    @password = password
+  end
+
+  def call
+    JsonWebToken.encode(user_id: user.id) if user
```

Comment:
double database access, we can easily avoid by initializing a local var to `user`

Reply:
addressed.